src/signals/incident/definitions/wizard-step-2-vulaan.js

```diff
@@ -17,7 +17,7 @@ export default {
   formAction: 'UPDATE_INCIDENT',
   formFactory: ({ category }) => {
     const noExtraProps = { controls: {} };
-    if(!configuration.hasExtraProps) return noExtraProps;
+    if (!configuration.showVulaanControls) return noExtraProps;
```

Comment:
The configuration object with throw an exception when a prop isn't there. Might be better to wrap this in a try/catch

Reply:
Added extra check but be aware that this extra check is not don for any of the configuration object properties...

Reply:
You're right. In the case of querying for props of the `configuration` object, we can actually do two things:

__try/catch__
```jsx
let nestedProp;
try {
  nestedProp = configuration.map.options.nestedProp;
} catch (e) {
 //
}
```

__conditional chaining__
```
const nestedProp = configuration?.map?.options?.nestedProp;
```

In your case, conditional chaining would be fine.